app.py

```diff
@@ -100,6 +102,43 @@ def get_members_by_state(state_):
     except Exception as e:
         return(str(e))
 
+@app.route("/users_reps/<user_id_>")
+def get_users_reps(user_id_):
+    user = User.query.filter_by(id=user_id_).first()
+    headers = {'X-API-Key': os.getenv('PROP_API')}
+    URL_SENATE = f'https://api.propublica.org/congress/v1/members/senate/{user.state}/current.json'
+    URL_HOUSE = f'https://api.propublica.org/congress/v1/members/house/{user.state}/{user.district}/current.json'
+    senate_response = requests.get(URL_SENATE, headers = headers).json()
+    house_response = requests.get(URL_HOUSE, headers = headers).json()
+    senate_results = senate_response['results']
```

Comment:
NICE! this will be a great feature, we just need to connect it with the FE

Reply:
Thanks! I am going to create a user story for the FE now.